addons/web-support/api/src/main/java/org/jboss/windup/web/addons/websupport/model/MigrationProjectModel.java

```diff
@@ -0,0 +1,59 @@
+package org.jboss.windup.web.addons.websupport.model;
+
+
+import com.tinkerpop.blueprints.Direction;
+import com.tinkerpop.frames.Adjacency;
+import com.tinkerpop.frames.Property;
+import org.jboss.windup.graph.model.WindupVertexFrame;
+
+/**
+ * A migration project is a group of applications which are related to each other
+ * and migrated as a bigger enterprise system.
+ *
+ * @author <a href="http://ondra.zizka.cz/">Ondrej Zizka, zizka@seznam.cz</a>
+ */
+public interface MigrationProjectModel extends WindupVertexFrame
+{
+    static final String TYPE_PROP = "MigrationProject";
+    static final String PROP_ID = TYPE_PROP + "-id";
+    static final String PROP_TITLE = TYPE_PROP + "-title";
+
+    /**
+     * The unique ID of this migration project.
+     */
+    @Property(PROP_ID)
+    String getId();
```

Comment:
We can't control the vertex ID IIRC. If the project would be dropped and recreated again, it couldn't have the same ID again.